src/Persistence/Initialization/Updates/FixAttackSpeedCalculationUpdate.cs

```diff
@@ -137,6 +137,7 @@ protected override async ValueTask ApplyAsync(IContext context, GameConfiguratio
         AddStatIfNotExists(context, gameConfiguration, Stats.WalkSpeed);
 
         Stats.AttackSpeed.GetPersistent(gameConfiguration).MaximumValue = Stats.AttackSpeed.MaximumValue;
+        Stats.MagicSpeed.GetPersistent(gameConfiguration).MaximumValue = Stats.MagicSpeed.MaximumValue;
```

Comment:
This will not work when someone has already run this update.
To make this work 100% correct, you'd need to add a new separate update plugin.
However, for such a small change, we can also ignore it.